yarn.lock

```diff
@@ -20812,6 +20850,15 @@ wrap-ansi@^6.2.0:
     string-width "^4.1.0"
     strip-ansi "^6.0.0"
 
+wrap-ansi@^7.0.0:
```

Comment:
🙈

Reply:
oh man, Yarn v1 really is running on its last breath in this repo 🙈